Neos.Eel/Classes/Validation/ExpressionSyntaxValidator.php

```diff
@@ -28,7 +28,9 @@ protected function isValid($value)
 
         if ($result === false) {
             $this->addError('Expression "%s" could not be parsed.', 1421940748, [$value]);
+            /** @phpstan-ignore-next-line */
```

Comment:
```suggestion
            /** @phpstan-ignore-next-line $parser->pos appears to be undeclared, because instantiation of $parser is hidden as well */
```

Again, here L25 would need treatment as well:
```php
/** @phpstan-ignore-next-line `EelParser` has been explicitly excluded from analysis, because it contains generated code that is impossible to fix */
```